guest-tools/image/create-td-image.sh

```diff
@@ -180,11 +180,30 @@ create_guest_image() {
     resize_guest_image
 }
 
+# To resize the guest image
+# 1) we add additionnal space to the qcow image using qemu-img tool
+# 2) we extend (using growpart) the partition sda1 to fill empty space until end of disk
+#    since sda1 is the last partition, it will take all space we previously added
+# 3) we resize the file system to cover all partition space
+#
+# NB: We should not use static name for the disk device (sda) because it can
+# change on boot (the main disk is named sdb). Using sda naming can cause failure
```

Comment:
Do you mean: "e.g., the main disk might be named sdb"?

Reply:
Yes

Reply:
```suggestion
# change on boot (e.g., the main disk might be named sdb). Using sda naming can cause failure
```
Then, I'd propose to be more explicit.